pkg/store/storepb/rpc.proto

```diff
@@ -42,9 +42,14 @@ enum StoreType {
 
 message InfoResponse {
   repeated Label labels = 1 [(gogoproto.nullable) = false];
-  int64 min_time        = 2;
-  int64 max_time        = 3;
-  StoreType storeType   = 4;
+  int64 min_time = 2;
```

Comment:
Our manual pretty indent is gone ;p

not a blocker though.

Reply:
I tried, but it started looking weirder and weirder. Let me know if it bothers you :)

Reply:
Nah, I think your time is way more precious to waste it on  manual linting (: